test/cell_test.rb

```diff
@@ -9,6 +9,6 @@ def test_can_add_style
     cell = row.cells.first
 
     cell.add_style b: true
-    assert_equal({ b: true }, cell.raw_style)
+    assert_equal({b: true}, cell.raw_style)
```

Comment:
Very minor nitpick... I think having spaces inside curly braces is a popular Ruby tradition. Rubocop (with default config) likes that too. There are many other Rubocop warnings here though which is all my fault :smile:  (most of this was written before I started using Rubocop).